### PR TITLE
Change repo references from dm-k8s-prod to new repos and update build and push for client and server images

### PR DESCRIPTION
**Pull request by jackgr**
New GPC projects `kubernetes-helm` and `kubernetes-charts` have been created using names that align with the Github repo names. This PR changes the repo references to use them.

/cc @sparkprime, @vaikas-google

### CODE REVIEW

**Comment by jackgr** [comment]
Added a commit that closes #378.

**Comment by jackgr** [comment]
Added a commit that closes #538.

**Comment by runseb** [comment]
tested this and deployment works.

two things:
1-the helm client is not pushed yet, so this will need to be done when merged.
2-I am not lgtm'ing this yet because I think @adamreese prefers specific semver tags on images rather than latest.

**Comment by jackgr** [comment]
Thanks, @runseb. Re: those two things:

1. The helm client is pushed and get-install.sh retrieves it. I did that from my clone.
1. Per my comment in #538 I'm also in favor of using semver tags for releases, and have left the env var mechanism in place as the way to implement that. However, semver tags are not practical for day to day development, since we would have to cut a tag every day or two, which would quickly become a maintenance burden.

**Comment by jackgr** [comment]
#557

**Comment by vaikas-google** [comment]
Can you update the change description to include the other changes that are part of this change. Otherwise LGTM

**Comment by jackgr** [comment]
Thanks, @vaikas-google. Done.

**Comment by jackgr** [comment]
Thanks, @sparkprime.

@runseb and @adamreese, if we want to institute semver default labeling for daily usage, we can submit a PR for that purpose.

File: pkg/client/install.go [hunk 1]
```diff
@@ -143,7 +143,7 @@ spec:
     spec:
       containers:
       - env: []
-        image: {{default "gcr.io/kubernetes-helm/expandybird:v1.2.1" .Expandybird.Image}}
+        image: {{default "gcr.io/kubernetes-helm/expandybird:latest" .Expandybird.Image}}
```
Comment (adamreese) [inline pkg/client/install.go]: Why are we going back to latest?

**Comment by adamreese** [comment]
why did you remove `.keep` files?  They had a purpose of maintaining the directory structure

**Comment by jackgr** [comment]
@adamreese the directory structure does not need to be maintained. It's now created on the fly. In addition, the bin directories are removed by `make clean`.

**Comment by jackgr** [comment]
@adamreese we're using `latest` not a semver label for day to day image pushing, since maintaining a set of labels on a day in day out basis would be a burden. By using latest, we always get what's at HEAD given the way the makefiles are set up now, which is generally what's needed for development. We still have the env var overrides for when we want to push a specific tag.

**Pull request merged**